.github/workflows/test_runner.yml

```diff
@@ -21,88 +21,12 @@ jobs:
-
-    - name: Setting up g3w-suite
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "/code/ci_scripts/setup_suite.sh"
-
-    - name: Starting Django
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py runserver 0.0.0.0:8000" &
-
-    - name: Waiting for Django
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "wait-for-it -h localhost -p 8000 -t 240"
-
-    - name:  Running g3w-admin core tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test core"
-
-    - name:  Running g3w-admin qdjango tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test qdjango"
-
-    - name:  Running g3w-admin usersmanage tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test usersmanage"
-
-    - name:  Running g3w-admin client tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test client"
-
-    - name:  Running g3w-admin editing tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test editing.tests"
-
-    - name:  Running g3w-admin caching tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test caching"
-
-    - name:  Running g3w-admin filemanager tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test filemanager"
-
-    - name:  Running g3w-admin qplotly tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test qplotly"
-
-    - name:  Running g3w-admin openrouteservice tests
-      run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test openrouteservice"
-
-    - name:  Running g3w-admin qtimeseries tests
+    - name:  Running g3w-admin tests
       run: |
-        docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml exec -T g3w-suite sh -c "cd /code/g3w-admin && python3 manage.py test qtimeseries"
+        make tests \
+          q=${{ matrix.qgis_version }} \
+          mode=test \
+          DOCKER_COMPOSE="docker-compose -f docker-compose.${{ matrix.qgis_version }}.yml"
```

Comment:
```
Docker Compose v1 1.29.2
Docker Compose v2 2.14.0+azure-1
```

This happens because some of these packets are available for [GitHub Actions](https://docs.github.com/en/actions) or [Azure DevOps](https://azure.microsoft.com/en-us/products/devops/) (that's why `Docker Compose v2 2.14.0+azure-1` is only available for Azure V1 pipelines)

Ref: [runner-images#faqs](https://github.com/actions/runner-images#faqs)

---

## [runner-images/images/linux/scripts/installers/docker-compose.sh](https://github.com/actions/runner-images/blob/b665bad57bcd15dfcdb4d951e164a0f9cc67e36f/images/linux/scripts/installers/docker-compose.sh#L1-L12)

```sh
#!/bin/bash -e
################################################################################
##  File:  docker-compose.sh
##  Desc:  Installs Docker Compose
################################################################################

# Install docker-compose v1 from releases
URL="https://github.com/docker/compose/releases/download/1.29.2/docker-compose-Linux-x86_64"
curl -L $URL -o /usr/local/bin/docker-compose
chmod +x /usr/local/bin/docker-compose

invoke_tests "Tools" "Docker-compose v1"
```

---

## [runner-images/images/win/scripts/Installers/Install-Docker.ps1](https://github.com/actions/runner-images/blob/6cc874356c0341862bd695e13db58ce7c1cc72e4/images/win/scripts/Installers/Install-Docker.ps1)

```ps
Write-Host "Install-Package Docker-Compose v1"
Choco-Install -PackageName docker-compose

Write-Host "Install-Package Docker-Compose v2"
$dockerComposev2Url = "https://github.com/docker/compose/releases/latest/download/docker-compose-windows-x86_64.exe"
$cliPluginsDir = "C:\ProgramData\docker\cli-plugins"
New-Item -Path $cliPluginsDir -ItemType Directory
Start-DownloadWithRetry -Url $dockerComposev2Url -Name docker-compose.exe -DownloadPath $cliPluginsDir
```